readthedocs/projects/tests/test_build_tasks.py

```diff
@@ -146,14 +148,15 @@ def test_build_sphinx_formats(self, load_yaml_config, formats, builders):
                 )
             )
 
-    @mock.patch('readthedocs.projects.tasks.builds.UpdateDocsTask.build_docs_html')
-    @mock.patch('readthedocs.projects.tasks.builds.UpdateDocsTask.build_docs_class')
     @mock.patch('readthedocs.projects.tasks.builds.load_yaml_config')
+    @mock.patch('readthedocs.projects.tasks.builds.UpdateDocsTask.build_docs_class')
+    @mock.patch('readthedocs.projects.tasks.builds.UpdateDocsTask.build_docs_html')
     def test_build_formats_only_html_for_external_versions(self, build_docs_html, build_docs_class, load_yaml_config):
         load_yaml_config.return_value = self._config_file({
             'version': 2,
             'formats': 'all',
         })
+        build_docs_html.return_value = True
```

Comment:
Why this mock requires a return value, but the following one (`test_build_respects_formats_mkdocs`, -which is almost identical) doesn't?

Reply:
Looks like isn't required anymore after fixing the api mocks.